web_src/less/_base.less

```diff
@@ -168,12 +168,9 @@
   --color-active-line: #fffbdd;
 }
 
-:root * {
-  --fonts-regular: var(--fonts-override, var(--fonts-proportional)), "Noto Sans", "Liberation Sans", sans-serif, var(--fonts-emoji);
-}
-
+/* There is a rule that sets --fonts-proportional on body in Fomantic UI */
 textarea {
-  font-family: var(--fonts-regular);
+  font-family: var(--fonts-proportional);
```

Comment:
Why not directly inside `:root` as you did in your other PR with the accent-color?
Where do we use different fonts?
I can't think of any case where we use another font…

Reply:
`font-family` does not inherit into `textarea` or `input`. We are mostly relying on Fomantic UI to set `font-family`, but this is the one exception.